src/plugins/displayLayout/DisplayLayoutToolbar.js

```diff
@@ -559,7 +559,8 @@ define(['lodash'], function (_) {
                 });
 
                 let toolbarArray = Object.values(toolbar);
-                return _.flatten(toolbarArray.reduce((accumulator, group, index) => {
+                const flatten = arr => arr.reduce((a, b) => a.concat(b), []);
```

Comment:
Can we implement this in a utils folder so we can use in other places as well?

Reply:
added flatten function to array-utils.js and imported for reuse